src/stampy_nlp/utilities/coda_utils.py

```diff
@@ -21,21 +21,30 @@
 RICH_TEXT_COLUMN = 'c-a82d-vxRc9'
 
 
-def get_df_data():
-    """Fetches questions from Coda and returns a pandas dataframe for processing"""
-    logging.debug("get_df_data()")
-
+def fetch_all_rows():
     headers: object = {'Authorization': f'Bearer {get_coda_token()}'}
     params: object = {
         'useColumnNames': True,
         'limit': 1000
     }
-    json_items = requests.get(TABLE_URL, headers=headers, params=params).json()['items']
+    response = requests.get(TABLE_URL, headers=headers, params=params).json()
+    while response:
```

Comment:
coda will not return more than 500 items - to get more the results have to be paginated :/

Reply:
I was getting back 500 results. Which is a suspiciously round number :D